posts/2022-04-07-Rust-1.60.0.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: "Announcing Rust 1.60.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.60.0. Rust is a programming language empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via rustup, you can get 1.60.0 with:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install]
+from the appropriate page on our website, and check out the
+[detailed release notes for 1.60.0][notes] on GitHub.
+If you'd like to help us out by testing future releases, you might consider updating locally to use
+the beta channel (`rustup default beta`) or the nightly channel (`rustup default nightly`). Please [report] any bugs you might come across!
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1600-2022-04-07
+[report]: https://github.com/rust-lang/rust/issues/new/choose
+
+## What's in 1.60.0 stable
+
+### Instrumentation-based Code Coverage
+
+Support for LLVM-based coverage instrumentation has been stabilized in rustc, enabled with `-C instrument-coverage`. You can try this out on your code by rebuilding your code with `-Cinstrument-coverage`:
+
+```shell=
+RUSTFLAGS="-C instrument-coverage" cargo build
+```
+
+After that, you can run the resulting binary, which will produce
+a `default.profraw` file in the current directory. That can be consumed
```

Comment:
Instead of the sentence starting "That can be consumed...", I suggest:

> The `llvm-tools-preview` component includes `llvm-profdata` for processing and merging raw profile output (coverage region execution counts); and `llvm-cov` for report generation. `llvm-cov` combines the processed output, from `llvm-profdata`, and the binary itself, because the binary embeds a mapping from counters to actual source code regions.

It's hard to make this explanation both terse and useful. You may want to word this differently.